zipline/assets/_assets.pyx

```diff
@@ -240,7 +241,8 @@ cdef class Future(Asset):
                   object expiration_date=None,
                   object first_traded=None,
                   object exchange="",
-                  int contract_multiplier=1):
+                  int contract_multiplier=1,
+                  object fuzzy=""):
```

Comment:
@jfkirk fwiw I've been re-thinking the fuzzy lookup change that I made. I think we may be able to support the interpolation at runtime, the 'fuzzy' value which was the symbol name with the fuzzy token dropped may be not worth the complexity/rigidity.